projects/core/components/scrollbar/scrollbar.options.ts

```diff
@@ -0,0 +1,22 @@
+import type {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk/utils/miscellaneous';
+
+export interface TuiScrollbarOptions {
+    showScrollbarWhenScrolling: boolean;
```

Comment:
This should be some kind of `mode` rather than a boolean. Also iOS shows bars when **scrolling**, not when hovering. I think showing it on hover is much better, just pointing out the naming inconsistency here.

Reply:
fixed